api/src/pcapi/core/users/external/zendesk_sell.py

```diff
@@ -148,7 +148,7 @@ def _get_venue_data(venue: offerers_models.Venue, parent_organization_id: int |
     params: dict = {
         "data": {
             "is_organization": True,
-            "name": venue.publicName if venue.publicName else venue.name,
+            # "name": venue.publicName if venue.publicName else venue.name,
```

Comment:
Plutôt que de laisser du **code** commenté (sans expliquer pourquoi), je suggère plutôt de laisser un commentaire indiquant pourquoi tel ou tel champ n'est pas mis à jour. 

```suggestion
            # "name" is not updated because XXX
```

Et je pense qu'il est important d'indiquer la raison, parce que le ticket Jira ne l'explique pas. Et quand on se posera la question dans 6 mois, on sera bien content de savoir pourquoi on ne met pas à jour `name`. :)


(idem plus bas)